napari/_vispy/_tests/test_vispy_vectors_layer.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import pytest
+
+from napari._vispy.layers.vectors import generate_vector_meshes, generate_vector_meshes_2D
+
+@pytest.mark.parametrize("edge_width, length, dims", [[0, 0, 2], [0.3, 0.3, 2], [1, 1, 3]])
```

Comment:
How important is it to parameterize `edge_width` and `length`? Should I keep these here?

Reply:
certainly not bad - maybe you could time and see how long whole test takes, if small then nice to have, if large then could maybe drop

Reply:
With default verbosity, this tests takes about 0.24s. With `-vvv` it takes a little over 1s.